test/e2e/index.js

```diff
@@ -107,7 +108,8 @@ if (typeof ReactDOM.unstable_createRoot === 'function') {
   const root = ReactDOM.unstable_createRoot(container);
   root.render(children);
 } else {
-  ReactDOM.render(children, container);
+  const root = createRoot(container);
```

Comment:
We can get rid of the condition and use only `createRoot`.

Reply:
Actually, nope, we can't do this, as we are still running tests with React 17 on our CI pipeline.